experiments/OceanBoxGCM/homogeneous_box.jl

```diff
@@ -91,11 +91,11 @@ function run_homogeneous_box(; imex::Bool = false, BC = nothing)
 
     result = ClimateMachine.invoke!(solver_config)
 
-    maxQ = Vars{vars_state_conservative(driver_config.bl, FT)}(maximum(
+    maxQ = Vars{vars_state(driver_config.bl, Conservative(), FT)}(maximum(
```

Comment:
why `Conservative()` and not `::Conservative`? What's the difference?

Reply:
`Conservative()` is just a singleton. The function calls need `Conservative()` and method definitions need `::Conservative`. Thanks for catching this!

Reply:
@kpamnany Is one preferred over the other? I think these should all inline anyway

Reply:
I don't know if one is preferred but would like to! @vchuravy: thoughts please?